main/src/main/java/org/mobilitydata/gtfsvalidator/validator/TimepointTimeValidator.java

```diff
@@ -93,7 +92,7 @@ private boolean isTimepoint(GtfsStopTime stopTime) {
     private final String specifiedField;
 
     StopTimeTimepointWithoutTimesNotice(GtfsStopTime stopTime, String specifiedField) {
-      super(SeverityLevel.WARNING);
```

Comment:
Just writing my thoughts, no need to change anything. The fact that changing a severity level doesn't require changing any tests feels a bit off, not sure if it's a good thing or not.

Reply:
I added that to the [list of points ](https://github.com/MobilityData/gtfs-validator/issues/1052)to discuss regarding how our unit tests are set up.